Src/Plugins/Security/SEC101.SecurePlaintextSecrets.json

```diff
@@ -46,6 +46,12 @@
           "ContentsRegex": "$SEC101/006.GitHubPat",
```

Comment:
Call this GitHubPatLegacy

Reply:
Done.

---
In reply to: [608158139](https://github.com/microsoft/sarif-pattern-matcher/pull/366#discussion_r608158139) [](ancestors = 608158139)